src/test/java/test/asserts/FakeRestAsserts.java

```diff
@@ -0,0 +1,33 @@
+package test.asserts;
+
+import data.models.CreateActivityResponse;
+import data.models.GetActivityResponse;
+import org.testng.Assert;
+import org.testng.asserts.SoftAssert;
+
+public class FakeRestAsserts {
```

Comment:
Isto i ovo, ActivitesAsserts i tako za svaku test klasu rasporedjujemo i asserte i providere, da nam bude lakse da nadjemo kada nam nesto treba i da to bude skladno smesteno u neku celinu